309thEDDGE/opal
# Opal 579 update test notebooks flag dask gpu

https://jira.il4.dso.mil/browse/OPAL-579

The purpose of these changes is to:
- Check if Dask is available, and if it is, run the Dask_Cluster_Benchmark_notebook.ipynb notebook
- Check if a GPU is available, and if it is, run the gpu-test-notebook.ipynb notebook

To test:
- Ensure (docker-compose) opal is deployed on GFE
- Run the test_all.bash script:
```
bash /home/jovyan/opal/devops-software/test_all.bash 
```
